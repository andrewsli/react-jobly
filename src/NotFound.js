import React, { Component } from 'react';

class NotFound extends Component {
  render() {
    return (
      <h1>You're in the wrong neighborhood</h1>
    );
  }
}

export default NotFound;
