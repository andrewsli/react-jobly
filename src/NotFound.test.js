import React from 'react';
import NotFound from './NotFound';
import toJson from "enzyme-to-json";
import { shallow, mount } from "enzyme";

it('renders without crashing using shallow', () => {
  shallow(<NotFound />)
});

it('renders without crashing using mount', () => {
  mount(<NotFound />)
});

it("matches snapshot", function () {
  let wrapper = shallow(<NotFound />);
  let serialized = toJson(wrapper);
  expect(serialized).toMatchSnapshot();
});