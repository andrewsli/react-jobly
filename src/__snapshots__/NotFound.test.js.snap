// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<h1>
  You're in the wrong neighborhood
</h1>
`;
